{
  "url": "https://api.github.com/repos/tensorflow/tensorflow/issues/32089",
  "repository_url": "https://api.github.com/repos/tensorflow/tensorflow",
  "labels_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/32089/labels{/name}",
  "comments_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/32089/comments",
  "events_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/32089/events",
  "html_url": "https://github.com/tensorflow/tensorflow/issues/32089",
  "id": 487114197,
  "node_id": "MDU6SXNzdWU0ODcxMTQxOTc=",
  "number": 32089,
  "title": "TF2.0 RC tf.keras.model.Evaluate has display bug",
  "user": {
    "login": "gaborchris",
    "id": 31912572,
    "node_id": "MDQ6VXNlcjMxOTEyNTcy",
    "avatar_url": "https://avatars.githubusercontent.com/u/31912572?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/gaborchris",
    "html_url": "https://github.com/gaborchris",
    "followers_url": "https://api.github.com/users/gaborchris/followers",
    "following_url": "https://api.github.com/users/gaborchris/following{/other_user}",
    "gists_url": "https://api.github.com/users/gaborchris/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/gaborchris/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/gaborchris/subscriptions",
    "organizations_url": "https://api.github.com/users/gaborchris/orgs",
    "repos_url": "https://api.github.com/users/gaborchris/repos",
    "events_url": "https://api.github.com/users/gaborchris/events{/privacy}",
    "received_events_url": "https://api.github.com/users/gaborchris/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [
    {
      "id": 404586594,
      "node_id": "MDU6TGFiZWw0MDQ1ODY1OTQ=",
      "url": "https://api.github.com/repos/tensorflow/tensorflow/labels/stat:awaiting%20tensorflower",
      "name": "stat:awaiting tensorflower",
      "color": "f4b400",
      "default": false,
      "description": "Status  - Awaiting response from tensorflower"
    },
    {
      "id": 473172988,
      "node_id": "MDU6TGFiZWw0NzMxNzI5ODg=",
      "url": "https://api.github.com/repos/tensorflow/tensorflow/labels/type:bug",
      "name": "type:bug",
      "color": "159b2e",
      "default": false,
      "description": "Bug"
    },
    {
      "id": 1097546578,
      "node_id": "MDU6TGFiZWwxMDk3NTQ2NTc4",
      "url": "https://api.github.com/repos/tensorflow/tensorflow/labels/comp:keras",
      "name": "comp:keras",
      "color": "0052cc",
      "default": false,
      "description": "Keras related issues"
    },
    {
      "id": 1160540362,
      "node_id": "MDU6TGFiZWwxMTYwNTQwMzYy",
      "url": "https://api.github.com/repos/tensorflow/tensorflow/labels/TF%202.0",
      "name": "TF 2.0",
      "color": "e79b38",
      "default": false,
      "description": "Issues relating to TensorFlow 2.0"
    }
  ],
  "state": "closed",
  "locked": false,
  "assignee": {
    "login": "rchao",
    "id": 6505863,
    "node_id": "MDQ6VXNlcjY1MDU4NjM=",
    "avatar_url": "https://avatars.githubusercontent.com/u/6505863?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/rchao",
    "html_url": "https://github.com/rchao",
    "followers_url": "https://api.github.com/users/rchao/followers",
    "following_url": "https://api.github.com/users/rchao/following{/other_user}",
    "gists_url": "https://api.github.com/users/rchao/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/rchao/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/rchao/subscriptions",
    "organizations_url": "https://api.github.com/users/rchao/orgs",
    "repos_url": "https://api.github.com/users/rchao/repos",
    "events_url": "https://api.github.com/users/rchao/events{/privacy}",
    "received_events_url": "https://api.github.com/users/rchao/received_events",
    "type": "User",
    "site_admin": false
  },
  "assignees": [
    {
      "login": "rchao",
      "id": 6505863,
      "node_id": "MDQ6VXNlcjY1MDU4NjM=",
      "avatar_url": "https://avatars.githubusercontent.com/u/6505863?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rchao",
      "html_url": "https://github.com/rchao",
      "followers_url": "https://api.github.com/users/rchao/followers",
      "following_url": "https://api.github.com/users/rchao/following{/other_user}",
      "gists_url": "https://api.github.com/users/rchao/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rchao/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rchao/subscriptions",
      "organizations_url": "https://api.github.com/users/rchao/orgs",
      "repos_url": "https://api.github.com/users/rchao/repos",
      "events_url": "https://api.github.com/users/rchao/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rchao/received_events",
      "type": "User",
      "site_admin": false
    }
  ],
  "milestone": null,
  "comments": 3,
  "created_at": "2019-08-29T18:36:20Z",
  "updated_at": "2020-04-29T23:24:12Z",
  "closed_at": "2019-11-11T20:07:18Z",
  "author_association": "NONE",
  "active_lock_reason": null,
  "body": "<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>\r\n\r\n**System information**\r\n- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No. This is the code from https://www.tensorflow.org/beta/tutorials/quickstart/beginner\r\n- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Google Colab and Ubuntu 18.04\r\n- TensorFlow version (use command below): pip install tensorflow==2.0.0-rc0\r\n- Python version: 3.6\r\n\r\n\r\n**Describe the current behavior**\r\n\r\nWhen I run model.evaluate with verbose=True, I get too many \"=======\" signs. I noticed this on Ubuntu 18.04 when I upgraded from TF2.0-beta to TF2.0.0-rc0. This is a mild annoyance when I am running models with soft-warp enabled on the command line.\r\n\r\n**Describe the expected behavior**\r\n\r\nThere should only be this many equal signs:\r\n[==============================]\r\nNot\r\n[===========================............................................\r\n\r\n**Code to reproduce the issue**\r\nProvide a reproducible test case that is the bare minimum necessary to generate the problem.\r\n``` from __future__ import absolute_import, division, print_function, unicode_literals\r\n\r\n# Install TensorFlow\r\ntry:\r\n  # %tensorflow_version only exists in Colab.\r\n  %tensorflow_version 2.x\r\nexcept Exception:\r\n  pass\r\n\r\nimport tensorflow as tf\r\n\r\nmnist = tf.keras.datasets.mnist\r\n\r\n(x_train, y_train), (x_test, y_test) = mnist.load_data()\r\nx_train, x_test = x_train / 255.0, x_test / 255.0\r\n\r\nmodel = tf.keras.models.Sequential([\r\n  tf.keras.layers.Flatten(input_shape=(28, 28)),\r\n  tf.keras.layers.Dense(128, activation='relu'),\r\n  tf.keras.layers.Dropout(0.2),\r\n  tf.keras.layers.Dense(10, activation='softmax')\r\n])\r\n\r\nmodel.compile(optimizer='adam',\r\n              loss='sparse_categorical_crossentropy',\r\n              metrics=['accuracy'])\r\nmodel.fit(x_train, y_train, epochs=1)\r\n\r\nmodel.evaluate(x_test, y_test)\r\n```\r\n\r\n**Other info / logs**\r\nThis issue may be due to the denominator of the number of evaluated samples completed. It should be 10000/10000, not 10000/1.\r\nInstead of 10000/1 [====\r\n",
  "closed_by": {
    "login": "jvishnuvardhan",
    "id": 46058173,
    "node_id": "MDQ6VXNlcjQ2MDU4MTcz",
    "avatar_url": "https://avatars.githubusercontent.com/u/46058173?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/jvishnuvardhan",
    "html_url": "https://github.com/jvishnuvardhan",
    "followers_url": "https://api.github.com/users/jvishnuvardhan/followers",
    "following_url": "https://api.github.com/users/jvishnuvardhan/following{/other_user}",
    "gists_url": "https://api.github.com/users/jvishnuvardhan/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/jvishnuvardhan/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/jvishnuvardhan/subscriptions",
    "organizations_url": "https://api.github.com/users/jvishnuvardhan/orgs",
    "repos_url": "https://api.github.com/users/jvishnuvardhan/repos",
    "events_url": "https://api.github.com/users/jvishnuvardhan/events{/privacy}",
    "received_events_url": "https://api.github.com/users/jvishnuvardhan/received_events",
    "type": "User",
    "site_admin": false
  },
  "reactions": {
    "url": "https://api.github.com/repos/tensorflow/tensorflow/issues/32089/reactions",
    "total_count": 0,
    "+1": 0,
    "-1": 0,
    "laugh": 0,
    "hooray": 0,
    "confused": 0,
    "heart": 0,
    "rocket": 0,
    "eyes": 0
  },
  "timeline_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/32089/timeline",
  "performed_via_github_app": null
}
